### ISSUE TITLE
Multiple Color Pickers In WP 3.5

### ISSUE THREAD

**Issue by cyberbitegroup** [bainternet/Admin-Page-Class#13]
When you have multiple color pickers in one tab it seems it's opening the wrong tab.  Here's a code chunk for the admin JS that replaces all the colorpicker JS you have now:

$('.at-color-select').click(function(e) {
    colorPicker = jQuery(this).next('div');
    input = jQuery(this).prev('input');

```
$.farbtastic($(colorPicker), function(a) { $(input).val(a).css('background', a); });

colorPicker.show();
e.preventDefault();

$(document).mousedown( function() { $(colorPicker).hide(); });
```

 });

**Comment by bainternet** [comment]
Once again can't replicate and its working just fine on my end 
tested in: 
WordPress 3.5 and WordPress multisite 3.5
share some more details, when is the wrong tab being opened? steps to duplicate?

**Comment by cyberbitegroup** [comment]
When I click the second or third color picker with the code you had in there, it was opening the first color picker box always.  I think it had something to do with using a conditional along with it.

I had 3 color pickers on a conditional and when I used the conditional with them, it wouldn't let me open anything but the first color picker so I used that code above and it seemed to fix the issue.

Note: Just tested and it doesn't seem to happen outside of a conditional, so maybe thats it.
